mainnet-dryrun-2/chain.json

```diff
@@ -0,0 +1,106 @@
+{
+  "$schema": "https://raw.githubusercontent.com/cosmos/chain-registry/master/chain.schema.json",
+  "chain_name": "archway",
+  "status": "live",
+  "network_type": "testnet",
+  "website": "https://archway.io/",
+  "pretty_name": "Archway",
+  "chain_id": "mainnet-dry-run-2",
+  "bech32_prefix": "arch",
```

Comment:
this should be archway

Reply:
Are you sure? Isn't the prefix `arch` ?
https://github.com/satoshilabs/slips/blob/master/slip-0173.md

Reply:
Yes it should be archway, we need to update the slip as well.